scripts/entry/app.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from typing import Dict
+import cfn_flip
+import json
+import subprocess
+import pathlib
+
+class Utils:
+    @staticmethod
+    def get_dict_from_template(filepath):
+        with open(filepath) as f:
+            template_raw = f.read()
+        template_flat_json = cfn_flip.to_json(template_raw)
+        template_dict = json.loads(template_flat_json)
+        return template_dict
+
+@dataclass
+class EntryTemplateFactory:
+    boilerplate: Dict
+    main:        Dict
+    entry:       Dict = None
+
+
+    @staticmethod
+    def new(**kwargs):
+
+        main_filepath = kwargs['main_filepath']
+
+        if 'boilerplate_filepath' in kwargs:
+            boilerplate_filepath = kwargs['boilerplate_filepath']
+        else:
+            boilerplate_filepath = 'input/boilerplate.yaml'
+
+        main_dict = Utils.get_dict_from_template(main_filepath)
+        boilerplate_dict = Utils.get_dict_from_template(boilerplate_filepath)
+
+
+        params = {
+            'boilerplate': boilerplate_dict,
+            'main': main_dict,
+        }
+
+        factory = EntryTemplateFactory(**params)
+        factory._update_entry(main_filepath)
+        return factory
+
+    def get_entry_template(self, **kwargs):
+
+        if 'entry_filepath' in kwargs:
+            entry_filepath = kwargs['entry_filepath']
+        else:
+            entry_filepath = 'output/entry.yaml'
+        entry_dict = self.entry
+        filename = pathlib.Path(entry_filepath).stem
+        path = pathlib.Path(entry_filepath).parent
+        with open(f"{path}/{filename}.json",'w') as f:
+            f.write(json.dumps(entry_dict))
+
+        command = f"cfn-flip {path}/{filename}.json {path}/{filename}.yaml"
+        subprocess.run(command, shell=True)
```

Comment:
Not sure I saw documented anywhere that user needs to install this system executable in order to run python script.

Reply:
I somehow had trouble getting it to work within Python so defaulted to the command-line way (which I first tried and got to work). I'll play around with it some more to see if it'll work for me